libs/NfcCoreLib/lib/Nci/phNciNfc.c

```diff
@@ -677,6 +677,70 @@ NFCSTATUS phNciNfc_Nfcee_ModeSet(void * pNciHandle,
     return wStatus;
 }
 
+NFCSTATUS phNciNfc_Nfcee_SePowerAndLinkCtrlSet(void * pNciHandle,
+                                               void * pNfceeHandle,
+                                               phNciNfc_PowerLinkModes_t eActivationMode,
+                                               pphNciNfc_IfNotificationCb_t pNotifyCb,
+                                               void *pContext)
+{
+    NFCSTATUS wStatus = NFCSTATUS_SUCCESS;
+    phNciNfc_Context_t * pNciContext = (phNciNfc_Context_t *)pNciHandle;
+    pphNciNfc_NfceeDeviceHandle_t pDevHandle = \
+        (pphNciNfc_NfceeDeviceHandle_t)pNfceeHandle;
+    uint8_t *pTargetInfo;
+
+    PH_LOG_NCI_FUNC_ENTRY();
+    if (NULL == pNciContext)
+    {
+        PH_LOG_NCI_CRIT_STR("Stack not initialized (phNciNfc_Nfcee_SePowerAndLinkCtrlSet)");
```

Comment:
The function name is included in the logs by default. You don't need to add it manually.

Reply:
ok